app/helpers/form_helper.rb

```diff
@@ -51,7 +53,8 @@ def us_states_territories_select_tag(options = {})
       form_input_error_messages('state', options)
   end
 
-  # rubocop:disable MethodLength, Style/WordArray
+  # rubocop:disable MethodLength,WordArray
```

Comment:
I just noticed that you removed the space between the different cops. Why is that? The Rubocop docs example includes a space, which is the convention when separating things with a comma. https://github.com/bbatsov/rubocop/blob/d676de0f89f4fe790203fad5d1661fba7e49735d/manual/configuration.md#disabling-cops-within-source-code